spec/mailers/claims/user_mailer_spec.rb

```diff
@@ -33,6 +42,9 @@
 
         Claim funding for mentor training team
       EMAIL
+
```

Comment:
I was getting spec failures from my expectations not matching /n to /r/n. Was unsure why they werent matching in this case and added a gsub to fix. Any ideas why this would be? Happy to remove it